WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController+JetpackPrompt.swift

```diff
@@ -28,9 +28,9 @@ extension NotificationsViewController {
             self?.activityIndicator.stopAnimating()
             self?.blogService.syncBlog(blog, success: {
                 self?.activityIndicator.stopAnimating()
-                if blog.account != nil {
-                    WPAppAnalytics.track(.signedInToJetpack, with: blog)
-                    WPAppAnalytics.track(.performedJetpackSignInFromNotificationsScreen, with: blog)
+                if blog.account != nil || !AccountHelper.isDotcomAvailable() {
```

Comment:
I think this needs to check the opposite (drop the `!`). I tested this and when I canceled the flow before login, I got a blank screen:

![simulator screen shot - iphone 8 - 2018-02-23 at 15 19 01](https://user-images.githubusercontent.com/8739/36598695-5423f7ae-18ad-11e8-8b86-34e509c46031.png)

When I canceled after login, I got the prompt again:

![simulator screen shot - iphone 8 - 2018-02-23 at 15 20 31](https://user-images.githubusercontent.com/8739/36598711-608144f2-18ad-11e8-9db1-6ae1076f5135.png)

In that case, if the right check is `blog.account != nil || AccountHelper.isDotcomAvailable()`, then `blog.account != nil` is redundant since if the blog has an account, the second part will always be true.